### ISSUE TITLE
margin seems to be ignored

### ISSUE THREAD

**Issue by max-vogler** [excaliburjs/excalibur-tiled#443]
### Steps to Reproduce
Import a Tiled map using a Tiled tileset loading a single image with a margin (e.g. 8 pixels of blank space around the tiles, don't confuse with spacing between the tiles).

### Expected Result
excalibur-tiled should load the tileset correctly. 

### Actual Result
excalibur-tiled loads the tiles in a garbled way, some are transparent, some are the wrong tiles.

### Environment

- browsers and versions: Chrome 114
- operating system: macOS 13
- Tiled version: 1.10, 1.09
- Excalibur versions: 0.27.0
- Tiled plugin versions: 0.27.0
- (anything else that may be relevant) 

### Current Workaround
Creating a new tileset in Tiled without margin works.

**Comment by eonarheim** [comment]
@max-vogler Thanks for the issue! Definitely unexpected

Is this the margin you're setting in Tiled?

<img width="269" alt="image" src="https://github.com/excaliburjs/excalibur-tiled/assets/612071/0bb5e220-49cf-4fbd-adef-de46094927ac">

**Comment by max-vogler** [comment]
Yes, exactly! Excuse the short bug report without code/reproduction. My problem was solved after re-doing the tileset without margin and I don't see [margin being used in the plugin code](https://github.com/search?q=repo%3Aexcaliburjs%2Fexcalibur-tiled%20margin&type=code).

**Comment by eonarheim** [comment]
Hopefully a quick fix 🤞 I distinctly remember writing this type of thing for spritesheets in the main repo

**Comment by max-vogler** [comment]
Hey thank you so much for the quick fix, love the fast turnaround time! I left a short comment in https://github.com/excaliburjs/excalibur-tiled/commit/0fe3031cfa37f16e42221519095bf84a1b00c9f4#r120139294, because I think there might still be a bug related to applying Tiled margin on the right and bottom side, which is not captured by Excalibur's originOffset.
